gpu/segmentation/include/pcl/gpu/segmentation/gpu_extract_clusters.h

```diff
@@ -131,6 +131,7 @@ namespace pcl
           */
         void extract (std::vector<pcl::PointIndices> &clusters);
 
+
```

Comment:
This should be removed?

Reply:
Yes, this is a good idea. I have done that.